atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/impl/SnapshotTransaction.java

```diff
@@ -196,7 +196,7 @@
 
     protected final ConcurrentMap<TableReference, ConcurrentNavigableMap<Cell, byte[]>> writesByTable =
             Maps.newConcurrentMap();
-    protected final ConflictDetectionManager conflictDetectionManager;
```

Comment:
Initially I extracted the ConflictDetectionManager interface, but that made the diff giant. So to ease the burden on whoever is touching the internals here, this works too.